apps/src/applab/applab.js

```diff
@@ -1240,6 +1242,10 @@ Applab.execute = function() {
   divApplab.firstChild.focus();
 
   Applab.running = true;
+  $('#headers').addClass('readonly');
+  $('#codeWorkspace').addClass('readonly');
+  // TODO - only if designing?
```

Comment:
Yeah, looks like we already call renderDesignWorkspace while executing this level, and everything works fine because the root element for that component is hidden.